src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,39 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({sender, body, timeStamp, onLike, onUnlike}) => {
+  const [buttonText, setButtonText] = useState('🤍');
+
+  const onLikeClick = () => {
+    if (buttonText === '🤍') {
+      setButtonText('❤️');
+      onLike();
+    } else {
+      setButtonText('🤍');
+      onUnlike();
+    }
+  };
 
-const ChatEntry = () => {
   return (
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time"><TimeStamp time ={timeStamp} /></p>
```

Comment:
```suggestion
        <p className="entry-time"><TimeStamp time={timeStamp} /></p>
```

Nice job using the provided Timestamp component here.

Convention is to not have spaces around = sign when passing props